mmpose/datasets/pipelines/bottom_up_transform.py

```diff
@@ -6,19 +8,68 @@
 from .shared_transform import Compose
 
 
+def get_warpmatrix(theta, size_input, size_dst, size_target):
+    """Calculate the transformation matrix under the constraint of unbiased.
+    Paper ref: Huang et al. The Devil is in the Details: Delving into Unbiased
+    Data Processing for Human Pose Estimation (CVPR 2020).
+
+    Args:
+        theta (float): Rotation angle in degrees.
+        size_input (np.ndarray): Size of input image [w, h].
+        size_dst (np.ndarray): Size of output image [w, h].
+        size_target (np.ndarray): Size of ROI in input plane [w, h].
+
+    Returns:
+        matrix (np.ndarray): A matrix for transformation.
+    """
+    size_target = size_target
+    theta = theta / 180.0 * math.pi
+    matrix = np.zeros((2, 3), dtype=np.float32)
+    scale_x = size_dst[0] / size_target[0]
+    scale_y = size_dst[1] / size_target[1]
+    matrix[0, 0] = math.cos(theta) * scale_x
+    matrix[0, 1] = -math.sin(theta) * scale_x
+    matrix[0, 2] = scale_x * (-0.5 * size_input[0] * math.cos(theta) +
+                              0.5 * size_input[1] * math.sin(theta) +
+                              0.5 * size_target[0])
+    matrix[1, 0] = math.sin(theta) * scale_y
+    matrix[1, 1] = math.cos(theta) * scale_y
+    matrix[1, 2] = scale_y * (-0.5 * size_input[0] * math.sin(theta) -
+                              0.5 * size_input[1] * math.cos(theta) +
+                              0.5 * size_target[1])
+    return matrix
+
+
+def affine_joints(joints, mat):
```

Comment:
affine is an adjective

Reply:
How about the name `transform_joints_affine`?

Reply:
this name is inherit from the origin mmpose project https://github.com/open-mmlab/mmpose/blob/master/mmpose/datasets/pipelines/bottom_up_transform.py in line260

Reply:
It was private function before, so the naming was more or less arbitrary.

Now it does not have `_` prefix

Reply:
point taken, we can imitate the function name cv2.warpAffine(),  we have warpAffineJoints/warpAffineKeypoints/warpAffine_Joints/warpAffine_Keypoints/?

Reply:
`warp_affine_joints`

Reply:
'warp_affine_joints' is adopted